src/archive_write.cpp

```diff
@@ -73,13 +73,15 @@ void rchive_write_close(Rconnection con) {
   struct archive* out;
   struct archive_entry* entry;
   in = archive_read_disk_new();
-#ifndef __MINGW32__
+#if defined(_WIN32)
```

Comment:
Is there a reason for changing this?

Reply:
a... maybe please check https://github.com/r-lib/archive/pull/86 first then let's go back to this one

Reply:
merged #86 to this one (so that #86 and #80 can be merged by one review round)

Reply:
@gaborcsardi "Allow edits and access to secrets by maintainers" if on, so feel free to push the minor flag and complete it...